utilities/test_suite/HOST/runTests.py

```diff
@@ -548,17 +550,22 @@ def rpp_test_suite_parser_and_validator():
     resultsInfo += "\n    - Total test cases including all subvariants REQUESTED = " + str(numLines)
     resultsInfo += "\n    - Total test cases including all subvariants PASSED = " + str(numPassed)
     f.write(resultsInfo)
-    df['Data_Augmentation_Type'] = augVariations
-    df['Achieved_Improvement_Percentage'] = achievedPerf
+    batchPDAugVariations = [s.replace('Tensor', 'BatchPD') for s in tensorAugVariations]
+    df['Tensor_Augmentation_Type'] = tensorAugVariations
+    df['BatchPD_Augmentation_Type'] = batchPDAugVariations
+    df['Performance Speedup (%)'] = achievedPerf
     df['Test_Result'] = status
     # Calculate the number of cases passed and failed
     passedCases = df['Test_Result'].eq('PASSED').sum()
     failedCases = df['Test_Result'].eq('FAILED').sum()
 
-    summary_row = {'Data_Augmentation_Type': pd.NA,
-                   'Achieved_Improvement_Percentage': pd.NA,
+    summary_row = {'BatchPD_Augmentation_Type': pd.NA,
+                   'Tensor_Augmentation_Type': pd.NA,
+                   'Performance Speedup (%)': pd.NA,
                    'Test_Result': f'Final Results of Tests: Passed: {passedCases}, Failed: {failedCases}'}
 
+    print("\n",tabulate(df, headers = 'keys', tablefmt = 'psql'))
```

Comment:
Can we just do a print(df) here? 
Or any simpler display that uses just pandas? Since tabulate is a new dependency and we'll have to add it in the CI for automated testing.

Reply:
Done